utils/full-rules-test.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Undo whitelisting from `rules-test` if possible
+
+# Change directory to git root; taken from ../test/test.sh
+if [ -n "$GIT_DIR" ]
```

Comment:
How is this file run? How does `$GIT_DIR` get set? Using something like  `git rev-parse --show-toplevel)` to get the toplevel might make it easier to run this file since it won't depend of mysterious global variables.

Reply:
I run this by calling `./utils/full-rules-test.sh` in the root directory. It is taken from https://github.com/EFForg/https-everywhere/blob/27f6f230a4f7c7fdff59f20c3642ef3343e2b859/test/rules.sh#L7 

and honestly I do not know where does this variable come from :smile: